File: CSharp/FizzBuzz/FizzBuzzer.cs
using System;
namespace CodebarTests
{

    public class FizzBuzzer
    {
        private readonly int _num;
        public string Message;

        public FizzBuzzer(int num)
        {
            _num = num;
        }


        public string FizzBuzz()
        {
            if (_num % 3 == 0 && _num % 5 == 0) Message = "Fizz Buzz";

            else if (_num % 3 == 0) Message = "Fizz";

            else if (_num % 5 == 0) Message = "Buzz";

            else Message = _num.ToString();
          

            return Message;
        }
    }
}

File: JavaScript/fizzBuzz/FizzBuzzer.js

// when a FizzBuzzer object gets instantiated, it will have a limit of nums to operate on from the start
class FizzBuzzer {
    constructor(numLimit) {
        this.numLimit = numLimit;
    };


    // method
     fizzBuzz(numLimit) {  

        let result = '';
        let i = 0;

        for(i ; i < numLimit; i++) {

            if(numLimit[i] % 3 == 0){
                result = 'Fizz';

                if(numLimit[i] % 5 == 0){
                    result = 'Fizz Buzz';
                }
            }
            else if(numLimit[i] % 5 == 0 ){
                result = 'Buzz';
            }
            return result;
        }
    }
};

module.exports = FizzBuzzer;